src/throttle/amoc_throttle_process.erl

```diff
@@ -143,19 +143,13 @@ maybe_stop_timer(#state{tref = undefined}) ->
     ok;
 maybe_stop_timer(#state{tref = TRef}) ->
     {ok, cancel} = timer:cancel(TRef),
-    consume_all_timer_ticks(delay_between_executions).
+    amoc_throttle_controller:consume_all_messages(delay_between_executions).
```

Comment:
revert this change, it introduces erronous impression that we do here something with amoc_throttle controller interface.

Reply:
So where would you put it? The code was duplicated, wanted to put it in some common helper.